common/reviews/api/rush-lib.api.md

```diff
@@ -300,11 +308,34 @@ export class IndividualVersionPolicy extends VersionPolicy {
 export interface _INpmOptionsJson extends IPackageManagerOptionsJsonBase {
 }
 
+// @beta
+export interface IOperationRunner {
+    executeAsync(context: IOperationRunnerContext): Promise<OperationStatus>;
```

Comment:
Would a plugin be allowed/expected to call this? That seems dangerous.

Reply:
Call it? No. However, a plugin may provide new `IOperationRunner` instances and need to _implement_ it, so the API needs to be public.